spec/features/slug_generation_spec.rb

```diff
@@ -34,6 +34,20 @@
         expect(find_field('Slug').value).to eq 'something'
         expect(find_field('Final URL').value).to eq "#{guide.topic.path}/something"
       end
+
+      it 'remembers that the slug was edited when coming back to edit it' do
+        topic_section = create(:topic_section)
+
+        slug = 'my-custom-slug'
+        guide = create(:guide, :with_draft_edition, slug: "#{topic_section.topic.path}/#{slug}")
+        topic_section.guides << guide
+
+        visit edit_guide_path(guide)
+
+        fill_in 'Title', with: 'A New Title'
+        expect(find_field('Slug').value).to eq slug
+        expect(find_field('Final URL').value).to eq "#{guide.topic.path}/#{slug}"
+      end
```

Comment:
I wrote a comment and deleted it as my internal pettiness alarm kicked in but just in case you like it..

I like using real/literal values in tests where I can and only use variables/interpolation where I have to. I find it easier to read and more resilient to regressions.. For this test I'd be tempted to..

```
topic = create(:topic, path: '/service-manual/topic')
topic_section = create(:topic_section, topic: topic)
```  

which would let me write assertions like..

```
expect(find_field('Final URL').value).to eq "/service-manual/topic/my-custom-slug"
```

It saves the reader some thinking in my opinion. The downside is that there is duplication of values in the test.. which personally doesn't bother me at all. Not an important thing so up to you!